app/models/email_template.rb

```diff
@@ -42,6 +43,8 @@ def users
       User.mentee.where.not(registration_token: nil)
     when :unregistered_mentors
       User.mentor.where.not(registration_token: nil)
+    when :daily_summary
+      User
```

Comment:
honestly i think we don't have to use email templates for that - will be much easier to use normal mailer.